xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -871,6 +879,8 @@ void CGUIWindowVideoBase::GetContextButtons(int itemNumber, CContextButtons &but
           buttons.Add(CONTEXT_BUTTON_PLAY_AND_QUEUE, 13412);
         else
           buttons.Add(CONTEXT_BUTTON_PLAY_ONLY_THIS, 13434);
+
+        buttons.Add(CONTEXT_BUTTON_PLAY_NEXT, 10008);      // Play next
```

Comment:
This isn't the correct place, see the description of the if.
https://github.com/xbmc/xbmc/blob/ed0a72307046cd17c33ad8f548a6de8276150e15/xbmc/video/windows/GUIWindowVideoBase.cpp#L865-L867

Reply:
Fair point.
Does moving it [here](https://github.com/xbmc/xbmc/blob/0f3197d73d32b8bff21388339350ee7e21f6fb8c/xbmc/video/windows/GUIWindowVideoBase.cpp#L847) sound good?

Reply:
Yes, that's the place I would have expected it.